.github/workflows/prbuild.yaml

```diff
@@ -14,9 +14,9 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
```

Comment:
Looks like this change is triggering some new lint errors that need to be looked into. Can either do that in this PR, or remove the version bump of golangci-lint from this PR and deal with that separately.

Reply:
what do you mean remove the version bump?

Reply:
You got it 👍